src/interfaces/forms.d.ts

```diff
@@ -110,6 +110,7 @@ export interface IInputFieldProps extends IInputProps {
 export interface ITextAreaProps
   extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
   error?: string;
+  ref?: React.Ref<HTMLTextAreaElement>;
```

Comment:
add `| React.Ref<React.Component<ITextAreaProps>>` - it's helpful when someone uses `styled` from `react-emotion` library